extensions/libarchive/BinFiles.h

```diff
@@ -154,7 +154,7 @@ class BinManager {
   void setMinEntries(uint32_t entries) {
     minEntries_ = {entries};
   }
-  void setBinAge(uint64_t age) {
+  void setBinAge(std::chrono::milliseconds age) {
     binAge_ = {age};
```

Comment:
this kind of assignment was used to protect against accidental narrowing conversions, since we use `std::chrono::milliseconds` now I think they are unnecessary

Reply:
changed it in https://github.com/martinzink/nifi-minifi-cpp/commit/78f8658fde802c73a5965dd734a544acbc37dca3#diff-495a1240beddf26f5b8b605419131e85beb584bc830c457c7db3cdb9cbcc92c1R158